cirq/ops/pauli_string.py

```diff
@@ -303,7 +303,9 @@ def _apply_unitary_(self, args: 'protocols.ApplyUnitaryArgs'):
     def expectation_from_wavefunction(self,
                                       state: np.ndarray,
                                       qubit_map: Mapping[raw_types.Qid, int],
-                                      atol: float = 1e-7) -> float:
+                                      atol: float = 1e-7,
```

Comment:
While you're at it, force keywords starting with `atol`.

Add `check_preconditions` to the `Args:` list.